aeneas/src/vst/MethodEnv.v3

```diff
@@ -871,6 +891,13 @@ class MethodEnv {
 			if (ttype == Void.TYPE) return expr.bind(null, VarBinding.Void, ttype);
 			return expr.bind(null, VarBinding.Type(ttype), ttype);
 		}
+		// lookup a struct
+		var structName = expr.ident.name.image;
+		for (i < verifier.file.structs.length) {
```

Comment:
There is a `HashMap` in the program so you can do one hashtable lookup here.

Reply:
It seems that MethodEnv don't have access to the Program, so should I leave this be?

Reply:
You can get to the program through `verifier.verifier.prog`.

Reply:
thank you! Lookup now uses the hashmap